.gitignore

```diff
@@ -12,6 +12,19 @@
 *.sdf
 *.sln
 *.suo
+Debug
+*A.cpp
+*A.h
+*A.inl
+*C.cpp
+*C.h
+*C.inl
+*S.cpp
+*S.h
```

Comment:
I've been avoiding doing this for OpenDDS since there are a bunch of false positives, for example TAO/tao/Muxed_TMS.*